src/main/kotlin/io/snyk/plugin/services/SnykCliService.kt

```diff
@@ -37,15 +37,12 @@ class SnykCliService(val project: Project) {
 
             val commands = buildCliCommandsList(applicationSettings)
 
-            val projectPath = project.basePath!!
+            val projectPath = project.basePath
+                ?: throw IllegalStateException("Scan should not be performed on Default project (with `null` project base dir)")
```

Comment:
@ArtsiomCh , this message will be propagated to users via notificaitions, right?